### ISSUE TITLE
Add scheduled messages in the server

### ISSUE THREAD

**Issue by krishnanunnir** [krishnanunnir/server_monitor_bot#4]
User can set messages that are sent to the bot at regular intervals

**Comment by herman-brits** [comment]
So the user would type something like "ls - - time 20" to execute the ls command every 20 seconds? Am I understanding?

**Comment by krishnanunnir** [comment]
Yes, something similar, but I think it would be better to configure this at the server-side to prevent someone abusing it.

**Comment by herman-brits** [comment]
Ok, I'm getting on this. Should be done in a day or two.

**Comment by krishnanunnir** [comment]
Thanks a lot for your contribution!
